docroot/modules/custom/sitenow_p2lb/sitenow_p2lb.module

```diff
@@ -585,8 +587,7 @@ function sitenow_p2lb_process_paragraph($paragraph, $node) {
         'use_more' => $more,
         'use_more_link_url' => $more_path,
         'exposed_filter_values' => [
-          // @todo Set these dynamically.
-          'field_tags_target_id' => [],
+          'field_tags_target_id' => $filter_tags,
```

Comment:
I'm wondering if we want to have some logic to ensure these are only getting set if they have values? Maybe something like:
```php
$exposed_filter_values = [];
if (!is_null($filter_tags)) {
  $exposed_filter_values['field_tags_target_id'] = $filter_tags
}
// ...
'exposed_filter_values' => $exposed_filter_values,
```